api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestPipeline.java

```diff
@@ -119,6 +119,7 @@ public void onStateChanged(int state) {
             Thread.sleep(300);
 
             assertEquals(NNStreamer.PIPELINE_STATE_PAUSED, mPipelineState);
+            Thread.sleep(100);
```

Comment:
Is 100ms enough always? or would it require dynamic-sleeping (e.g.,  https://github.com/nnsuite/nnstreamer/blob/3095de8e4e6596d80509c06139517cc31790ea2f/tests/tizen_capi/unittest_tizen_capi.cpp#L130 )

Reply:
IMO, 100ms is enough in this line. This is to prevent the pipeline close for a while in the end of the loop.